pkg/analytics/analytics.go

```diff
@@ -0,0 +1,56 @@
+package analytics
+
+import (
+	"os"
+
+	v1 "github.com/mjpitz/go-ga/client/v1"
+	"github.com/mjpitz/go-ga/client/v1/gatypes"
+
+	"github.com/kubeshop/testkube/cmd/tools/commands"
+	"github.com/kubeshop/testkube/pkg/telemetry"
+)
+
+const testkubeTrackingID = "UA-221444687-1"
```

Comment:
Might be to pass it from env var?

Reply:
But it would be quite hard to force user to set it :)

Reply:
What will be the best solution?

Reply:
If I remember it right, we added config for kubectl plugin, so it can be store and read it from there for cli
For server side, we should be able to pass it through dashboard by api method call or siimilar from cli api call. The question is where to store it on server side and to keep it there, even if pod is destroyed

Reply:
Yeah will be a good place 
Someone could Use it's company if if he wants ;)